lib/client.ts

```diff
@@ -36,8 +38,13 @@ function pickJSON(bigint: boolean): typeof JSON | typeof JSONbig {
  * @api private
  */
 
-function handleJSONResponse(request, bigint, callback) {
-  return function onJSONResponse(response) {
+function handleJSONResponse(
```

Comment:
Bummer. I just inlined this function in the code that I'm working on now...

Either I need to rebase, or I need to include your changes in the work I'm doing now, or you would have to wait a bit and redo this (sorry...)

Reply:
I'll drop this part altogether and skip test for now :)